processing/data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, datetime, timedelta
+
+from scrapy import Request, Spider
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
```

Comment:
We need a rebase for it to work, @cuducos. If you want, I can push your branch up to date.
I'm still running the spider, I'll keep you posted about the results.